### ISSUE TITLE
Formatting enum comparison fails

### ISSUE THREAD

**Issue by rmaz** [nicklockwood/SwiftFormat#544]
Formatting the example:

```
XCTAssertFalse(.never < .never)
```

fails with:

```
warning: Failed to terminate.
error: Failed to format any files.
```

Other operators format correctly, only seems to occur with < or >.

**Comment by nicklockwood** [comment]
@rmaz thanks - investigating now.

**Comment by nicklockwood** [comment]
@rmaz it's a bug in the `yodaConditions` rule. You can fix it for now by temporarily disabling that rule.

**Comment by rmaz** [comment]
Thanks for the workaround

**Comment by nicklockwood** [comment]
@rmaz fixed in 0.44.1.

**Comment by rmaz** [comment]
👍
